sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -29,13 +38,40 @@ export interface LinkEntityOptions {
   audience?: string;
 }
 
+/**
+ * A simple grouping of the sender and receiver options. Only used
+ * with the ManagementClient today.
+ *
+ * @internal
+ * @ignore
+ */
+export interface RequestResponseLinkOptions {
+  senderOptions: SenderOptions;
+  receiverOptions: ReceiverOptions;
+  name?: string;
+}
+
+/**
+ * @internal
+ * @ignore
+ */
+type LinkOptionsT<
+  LinkT extends Receiver | AwaitableSender | RequestResponseLink
+> = LinkT extends Receiver
+  ? ReceiverOptionsWithSession
```

Comment:
`ReceiverOptionsWithSession` should be used if we want to create an AMQP receiver link in the given AMQP session. We don't do that in Service Bus. Every AMQP link gets to have its own AMQP session. So, this should be `ReceiverOptions`